Analyse.cs

```diff
@@ -3,34 +3,149 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using CMP1903M_Assessment_1.Debugging;
 
 namespace CMP1903M_Assessment_1
 {
+    /// <summary>
+    /// Stores and analyses the user input
+    /// </summary>
     public class Analyse
     {
-        //Handles the analysis of text
+        private string _text = string.Empty;
+        public string Text
+        {
+            get { return _text; }
+            private set { _text = value; }
+        }
+
+        public Analyse(string text)
+        {
+            Text = text;
+        }
+
+        readonly char[] Vowels = { 'a', 'e', 'i', 'o', 'u' };
+
+        /*
+        0. Sentence Count
+        1. Vowels Count
+        2. Sonsonants Count
+        3. Upper Case Letters Count
+        4. Lower Case Letters Count
+        */
+        public int[] Statistics = new int[5];
 
-        //Method: analyseText
-        //Arguments: string
-        //Returns: list of integers
-        //Calculates and returns an analysis of the text
-        public List<int> AnalyseText(string input)
+        public int[] LetterFrequencies = new int[26]; // add 97 to the index to get the char value
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        public int[] AnalyseText()

```

Comment:
I'd recommend researching [Regex](https://docs.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-language-quick-reference) here.
They make this much, much more concise and are very powerful.
Rather annoying to learn, but you'll be glad you did learn it.